src/ansys_sphinx_theme/__init__.py

```diff
@@ -34,13 +34,7 @@
 from ansys_sphinx_theme.extension.linkcode import DOMAIN_KEYS, sphinx_linkcode_resolve
 from ansys_sphinx_theme.latex import generate_404  # noqa: F401
 
-try:
-    import importlib.metadata as importlib_metadata
-except ModuleNotFoundError:  # pragma: no cover
-    import importlib_metadata
-
-__version__ = importlib_metadata.version(__name__.replace(".", "-"))
-
+__version__ = "0.16.dev0"
```

Comment:
Why are we changing the version approach?

Reply:
In updating the documentation for the dependencies, I've removed the importlib metadata dependencies when utilizing dynamic in pyproject.toml.  If there are any concerns or disagreements regarding this adjustment, I'm open to reverting these changes.

Reply:
Starting on Python 3.9, importlib.metadata comes by default with Python - which for our case means that you can drop the try/except and just perform the import. I would leave it as it was IMO, without the try/except. All repos are working the same way, plus I think it's cleaner to have the version in the project's configuration file rather than inside the source code itself.

Reply:
Also - in terms of scope - the title of the PR does NOT reflect this change so, in any case I'd do it in a different PR.

Reply:
ok noted, thank you and reverted the changes.